README.md

```diff
@@ -30,6 +30,14 @@ You can use the `--exclude-region` flag to exclude resources in certain regions
 aws-nuke --exclude-region ap-south-1 --exclude-region ap-south-2
 ```
 
+### Excluding Resources by Age
+
+You can use the `--exclude-after` flag to exclude resources created after a certain time, any resources older than that time will be deleted. For example the following command does not nuke resources created after the midnight on 1st Jan, 2017:
+
+```shell
+aws-nuke --exclude-after '01-01-2017 12:00AM'
```

Comment:
Would supporting a duration make more sense? I'm imagining a CI job that would always want to delete all resources older than, say, a day, rather than having to manually compute the date each time.

Reply:
I thought of this approach but it was hard for me to come up with a good syntax approach to that. For example if we had an `--older-than` flag what would the smallest unit be? days, minutes, seconds? Regardless of the one we go with there's gonna be some math involved if the user wants to ignore within a time period which I believe is more cognitive load than just looking up a date. More thoughts on this are welcome

Reply:
I actually like @josh-padnick's idea quite a bit. You can use syntax like `1d`, `5h`. This can be parsed using: https://golang.org/pkg/time/#ParseDuration